connector/server/handlers/medco_node.go

```diff
@@ -16,6 +16,34 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// MedCoNodeExploreSearchHandler handles the /medco/node/explore/search API endpoint.
+func MedCoNodeExploreSearchHandler(params medco_node.ExploreSearchParams, principal *models.User) middleware.Responder {
+
+	// set default value for limit if not present
+	if params.SearchRequest.Limit == nil {
+		params.SearchRequest.Limit = new(int64)
+		*params.SearchRequest.Limit = 10
```

Comment:
Would it be better to set default values in swagger definition YAML ?

Reply:
It is set, but default values do not work with nested properties in go swagger, that's why I put this here. But I found a nicer workaround.